### PR TITLE
Db/sqlsetup

### PR DESCRIPTION
**Pull request by arjun0419**
https://trello.com/c/aLsJF9gI/5-choose-secondary-db

### CODE REVIEW

File: database/review.js [hunk 1]
```diff
@@ -4,7 +4,17 @@ require('dotenv').config();
 // mongoose.connect(process.env[process.env.NODE_ENV]);
```
Comment (ZeMunchkin) [inline database/review.js]: Same comments about commented-out code

File: hello.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+config:
```
Comment (ZeMunchkin) [inline hello.yml]: Did you mean to commit this file? Do you need it with the repo?

File: knexfile.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    },
+    migrations: {
+      directory: `${__dirname}/database/knex/migrations`,
+    },
+    seeds: {
+      directory: `${__dirname}/database/knex/seeds`,
+    },
+  },
+
+  // staging: {
```
Comment (ZeMunchkin) [inline knexfile.js]: Commented code

File: newrelic.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+'use strict'
```
Comment (ZeMunchkin) [inline newrelic.js]: Definitely need to ignore this file since it has your newrelic key in it

File: newrelic_agent.log [hunk 5]
```diff
@@ -0,0 +1,757 @@
+{"v":0,"level":30,"name":"newrelic","hostname":"Arjun.local","pid":16654,"time":"2018-03-20T21:38:55.868Z","msg":"Using New Relic for Node.js. Agent version: 3.2.0; Node version: v8.9.3."}
```
Comment (ZeMunchkin) [inline newrelic_agent.log]: I'd ignore this file too

File: redisIndex.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+redisClient.on('error', (err) => {
+  console.log('Something went wrong ', err);
+});
+
+redisClient.on('connect', function() {
+    console.log('connected to redis');
+});
+
+module.exports.redisClient = redisClient;
+// client.set('my test key', 'my test value', redis.print);
```
Comment (ZeMunchkin) [inline redisIndex.js]: commented out code

File: server/nodeServerHelpers.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
```
Comment (ZeMunchkin) [inline server/nodeServerHelpers.js]: This is really clean! Nice! :)

File: server/nodeServerHelpers.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+          response.end();
+        } else {
+          response.writeHead(200, { 'Content-Type': 'application/json' });
+          redisClient.setex(`${reqId}`, 10, JSON.stringify(data));
+          // console.log('wrote to redis: ', reqId);
+          response.end(JSON.stringify(data));
+        }
+      });
+    } else {
+      // console.log('got from redis: ', reqId);
```
Comment (ZeMunchkin) [inline server/nodeServerHelpers.js]: take out the console.log

File: server/routes/routes.js [hunk 9]
```diff
@@ -1,11 +1,14 @@
 const express = require('express');
+// const db = require('../../database/psqlReview.js');
```
Comment (ZeMunchkin) [inline server/routes/routes.js]: commented out code

File: database/knex/migrations/20180317122716_setup.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    knex.schema.createTable('users', (table) => {
+      table.integer('user_id').primary();
+      table.string('username');
+      table.string('user_location');
+      table.integer('user_friends');
+      table.string('user_photo');
+    }),
+    knex.schema.createTable('reviews', (table) => {
+      table.integer('review_id').primary();
+      // table.integer('user_id').unsigned();
```
[the remark this answers is not in the document]
Comment (arjun0419) [reply database/knex/migrations/20180317122716_setup.js]: deleted

File: database/psqlReview.js [hunk 11]
```diff
@@ -0,0 +1,113 @@
+const { Pool, Client } = require('pg');
+require('dotenv').config();
+
+const client = new Client({
+  user: 'aj2016',
```
[the remark this answers is not in the document]
Comment (arjun0419) [reply database/psqlReview.js]: noted for future ref

File: database/review.js (hunk 1 above)
Comment (arjun0419) [reply database/review.js]: done

File: hello.yml (hunk 2 above)
Comment (arjun0419) [reply hello.yml]: deleted

File: knexfile.js (hunk 3 above)
Comment (arjun0419) [reply knexfile.js]: done

**Comment by arjun0419** [comment]
@ZeMunchkin refactored as per your comments. Will create env variable files on the next round. Thanks!

File: server/nodeServerHelpers.js (hunk 7 above)
Comment (arjun0419) [reply server/nodeServerHelpers.js]: :)

File: database/review.js [hunk 12]
```diff
@@ -27,9 +27,6 @@ const insertReview = (restaurantID, reviewObject, callback) => {
       }
     },
   );
-
-  // Restaurant.findOne({ id: restaurantID })
-  // .then((data)=>console.log(data));
```
Comment (ZeMunchkin) [inline database/review.js]: May want to take this out before merging

File: webpack.config.js [hunk 13]
```diff
@@ -4,22 +4,59 @@ const path = require('path');
...
+  entry: `${APP_DIR}/server-index.jsx`,
+  target: 'node',
+  output: {
+    path: BUILD_DIR,
+    filename: 'server-bundle.js',
+    libraryTarget: 'commonjs-module',
+  },
+};
+
+// const config = {
```
Comment (ZeMunchkin) [inline webpack.config.js]: commented out code

**Pull request merged**